reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/ManualTrackingRumTest.kt

```diff
@@ -395,6 +399,217 @@ class ManualTrackingRumTest {
 
     // endregion
 
+    // region AddViewLoading time
+
+    @OptIn(ExperimentalRumApi::class)
+    @RepeatedTest(16)
+    fun `M attach view loading time W addViewLoadingTime { active view available }`(
+        @StringForgery key: String,
+        @StringForgery name: String,
+        @BoolForgery overwrite: Boolean
+    ) {
+        // Given
+        val rumMonitor = GlobalRumMonitor.get(stubSdkCore)
+        val startTime = System.nanoTime()
+        rumMonitor.startView(key, name, emptyMap())
+
+        // When
+        val endTime = System.nanoTime()
+        val expectedViewLoadingTime = endTime - startTime
+        rumMonitor.addViewLoadingTime(overwrite)
+
+        // Then
+        val eventsWritten = stubSdkCore.eventsWritten(Feature.RUM_FEATURE_NAME)
+        StubEventsAssert.assertThat(eventsWritten)
+            .hasSize(2)
+            .hasRumEvent(index = 0) {
+                // Initial view
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(key)
+                hasViewName(name)
+                hasActionCount(0)
+            }
+            .hasRumEvent(index = 1) {
+                // view updated with loading time
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(key)
+                hasViewName(name)
+                hasViewLoadingTime(
+                    expectedViewLoadingTime,
+                    offset = Offset.offset(TimeUnit.MILLISECONDS.toNanos(5))
+                )
+            }
+    }
+
+    @OptIn(ExperimentalRumApi::class)
+    @RepeatedTest(16)
```

Comment:
same question as above (and same for the tests below)

Reply:
not sure, I just followed the pattern, @xgouchet do you have more input ?